src/Illuminate/Auth/Console/stubs/make/routes.stub

```diff
@@ -1,4 +1,4 @@
 
 Auth::routes();
 
-Route::get('/home', 'HomeController@index')->name('home');
+Route::get('/', 'HomeController@index')->name('home');
```

Comment:
This will conflict with the base laravel skeleton which already has a `/` route defined.

Reply:
Check what i have said in here  laravel/laravel#4819 #26134